includes/transformer/class-post.php

```diff
@@ -729,33 +729,18 @@ protected function get_type() {
 			return 'Note';
 		}
 
-		// Default to Article.
-		$object_type = 'Article';
-		$post_format = 'standard';
+		// Default to Note.
+		$object_type = 'Note';
+		$post_type   = \get_post_type( $this->wp_object );
 
-		if ( \get_theme_support( 'post-formats' ) ) {
+		if ( 'page' === $post_type ) {
+			$object_type = 'Page';
+		} else {
 			$post_format = \get_post_format( $this->wp_object );
-		}
 
-		$post_type = \get_post_type( $this->wp_object );
-		switch ( $post_type ) {
-			case 'post':
-				switch ( $post_format ) {
-					case 'standard':
-					case '':
-						$object_type = 'Article';
-						break;
-					default:
-						$object_type = 'Note';
-						break;
-				}
-				break;
-			case 'page':
-				$object_type = 'Page';
-				break;
-			default:
+			if ( ! $post_format ) {
```

Comment:
This is true for post types that don't support post formats. Is that the intention?
As opposed to post types that support post formats but the object doesn't have one assigned.

Reply:
this is for both... either they do not support post formats or the post format is "standard"...